content/updates/2020-07/consistency-in-structure.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/data-quality-attribute-grid.png
+pim_announcement_alt_img: "Data quality in attribute grid"
+pim_announcement_audience:
+- EE
+---
+
+# Avoid spelling mistakes in your product data
+::: meta-data type="Improvement" features="Data Quality Insights" available="late July" link-to-doc="../articles/improve-data-quality.html#improve-data-quality"
+:::
+
+You can now benefit from spell-checking on attributes and option labels to guarantee consistency on your customer-facing product data.
+
+You can now check and correct spelling mistakes in attributes in the `Settings` pages.
+This improvement is also available in the product form. New recommendations are listed should we find errors. The better the content, the higher the consistency grade! :wink:
+
+
+![dqi-tab-consistency-in-structure](../img/data-quality-spelling-option.png)
+
+::: more
+[What is data quality?](../articles/understand-data-quality.html)
```

Comment:
```suggestion
[What is data quality?](../articles/understand-data-quality.html)  
```
We need to add two spaces at the end of each line to force going to the next line.